dev/docker/kerberos-hive/krb5formac.conf

```diff
@@ -0,0 +1,19 @@
+[[logging]
+default = FILE:/var/log/krb5libs.log
+kdc = FILE:/var/log/krb5kdc.log
+admin_server = FILE:/var/log/kadmind.log
+
+[libdefaults]
+default_realm = HADOOPKRB
+default_ccache_name = FILE:/tmp/krb5cc_cli_%{uid}
+dns_lookup_realm = false
+dns_lookup_kdc = false
+ticket_lifetime = 24h
+renew_lifetime = 7d
+forwardable = true
+
+[realms]
+HADOOPKRB = {
+  kdc = tcp/localhost:88
+  admin_server = localhost
+}
```

Comment:
what is the usage of this file?

Reply:
Define the address and port of the realm of `HADOOPKRB`. This is a general practice

Reply:
I'm what is the usage of this **"FILE"**, you have a similar file "krb5.conf".

Reply:
As explained by @qqqttt123 , `krb5formac.conf` is used as the configuration files ITs to connect KDC installed the docker and need to be copied to the host computer, however `krb5.conf` is used as the files for the KDC server as the configuration files.

Reply:
<img width="1192" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/8229e261-25bb-4b0c-ba66-a5e2dd1b8d33">

Reply:
Add more comments on this. Also in which scenario do you need to copy this conf to the host machine?